packages/bug-detectors/command-injection.ts

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2023 Code Intelligence GmbH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+	BugDetectorError,
+	hookBuiltInFunction,
+	saveFirstBugDetectorError,
+} from "./index";
+import { nextFakePC } from "@jazzer.js/instrumentor/dist/plugins/helpers";
+import { guideTowardsEquality } from "@jazzer.js/fuzzer";
+
+// eslint-disable-next-line @typescript-eslint/ban-types
+export async function registerCommandInjectionBugDetectors<F extends Function>(
```

Comment:
ChatGPT add a description what this bug detector is doing :smile:

Reply:
Function name says it clearly! (that's what ChatGPT told me :wink: )